docs/program/extend/SDK-as-server.md

```diff
@@ -11,41 +11,61 @@ videoAlt: "A quadrupedal robot comprised of small servos, black laser cut acryli
 ---
 
 {{% alert title="Caution" color="caution" %}}
-Support for this method of creating a custom resource implementation will be removed in favor of the module system.
+Modular resources are the preferred method of creating custom resource implementations for SDKs with module support unless you are hosting `viam-server` on a non-Linux platform or have another issue with compilation.
+
 Instructions on creating and using modular resources are available [here](/program/extend/modular-resources).
 {{% /alert %}}
 
-If a type or model of [component](/components) you are working with is not built into the [Viam RDK](/internals/rdk), you can use a [Viam SDK](/program/sdk-as-client) to code a custom resource implementation, host it on a server, and add it as a [remote](/manage/parts-and-remotes) of your robot.
-Then, you can configure a command to launch this remote server as a [process](/appendix/glossary/#term-process) of your robot to make sure the remote server is always running alongside the rest of your robot.
+If a type or model of [component](/components) you are working with is not built-in to the [Viam RDK](/internals/rdk), you can use a [Viam SDK](/program/sdk-as-client) to code a custom resource implementation, host it on a server, and add it as a [remote](/manage/parts-and-remotes) of your robot.
```

Comment:
Link is probably wrong. THe "sdk-as-client" functionality should probably be sdk-as-server (which is actually where we're already at I think.)

Reply:
Oh fair enough the names of this pair of pages are not a great description, I don't know if we should change them within this pr but @npentrel do you think it would be a good idea to rename the main SDK page? sdk-as-client is the SDK description & install page which is why I linked